pkg/controller/cheinstallation/cheinstallation_controller_test.go

```diff
@@ -30,101 +30,84 @@ import (
 )
 
 func TestCheInstallationController(t *testing.T) {
-	t.Run("should reconcile with che installation", func(t *testing.T) {
+	t.Run("should reconcile with che installation and create che ns", func(t *testing.T) {
```

Comment:
the 3 subtests below depend on each other, right?

Reply:
@xcoulon All test case are independent due to the fact that .. each test case create new_instance of che_installation and each test case call `Reconcile()`  only one and verify the result.  So to ans your que, No sub-tests are not dependent on each other.